legend-engine-xt-relationalStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/StreamResultToTempTableVisitor.java

```diff
@@ -78,67 +71,23 @@ public StreamResultToTempTableVisitor(RelationalExecutionConfiguration config, C
     }
 
     @Override
-    public Boolean visit(RelationalDatabaseCommands databaseCommands)
```

Comment:
This looks incorrect. In fact, this file should not be a part of the review. Take a look at how we implement the TempTable strategy for sqlServer [here](https://github.com/finos/legend-engine/blob/master/legend-engine-xt-relationalStore-sqlserver-execution/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/SqlServerConnectionExtension.java)
Follow the same for spanner

Reply:
ok. Reverted